src/daemons/StorageDaemon.cpp

```diff
@@ -88,7 +91,15 @@ int main(int argc, char *argv[]) {
     uint32_t localIP;
     CHECK(NetworkUtils::ipv4ToInt(result.value(), localIP));
 
-    auto metaClient = std::make_unique<nebula::meta::MetaClient>();
+    if (FLAGS_meta_server_addrs.empty()) {
+        LOG(ERROR) << "meta_server_addrs flag should be set!";
+        return -1;
+    }
+    auto ioThreadPool = std::make_shared<folly::IOThreadPoolExecutor>(FLAGS_io_handlers);
```

Comment:
Could we share the same pool with the hosting service?

Reply:
Hosting service?  If it means the thrift service,  yes of course.  Please note line 143.